aten/src/ATen/native/native_functions.yaml

```diff
@@ -289,6 +289,7 @@
 - func: conj.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
   dispatch:
     CPU, CUDA: conj_out
+    SparseCPU, SparseCUDA: conj_out_sparse
```

Comment:
bloop. conjugate views gonna be hard on sparse but I guess we will figure something out. https://github.com/pytorch/pytorch/pull/44799